### PR TITLE
Docs: fix typo in quick start example

### PR DESCRIPTION
**Pull request by gleon99**

### CODE REVIEW

**Comment by hoopoepg** [comment]
ok to test

**Review by hoopoepg — APPROVED** [review]

**Review by yosefe — APPROVED** [review]

**Review by dmitrygx — APPROVED** [review]

**Review by brminich — APPROVED** [review]

**Comment by shamisp** [comment]
Thanks for the patch @gleon99. Are you affiliated with Nvidia ? thanks

**Comment by gleon99** [comment]
bot:pipe:retest

**Comment by gleon99** [comment]
@shamisp  yes

**Comment by brminich** [comment]
bot:pipe:retest

File: docs/source/index.rst [hunk 1]
```diff
@@ -58,7 +58,7 @@ and run a simple client/server example:
 
     $ gcc examples/ucp_client_server.c -lucp -lucs -o ucp_client_server \
           -Iinstall/include -Linstall/lib
-    $ export LD_LIBRARY_PATH=$PWD/instal/lib
+    $ export LD_LIBRARY_PATH=$PWD/install/lib
```
Comment (mike-dubman) [inline docs/source/index.rst:61]: i think lines 62-63 should start from ```./bin/ucp_client_server```

Comment (yosefe) [reply docs/source/index.rst:61]: it will not work, since line 59 writes the output binary in current directory

Comment (yosefe) [reply docs/source/index.rst:61]: @mike-dubman ok to merge?

Comment (mike-dubman) [reply docs/source/index.rst:61]: +1

**Pull request merged**